themes-default/slim/src/components/helpers/app-link.vue

```diff
@@ -124,7 +124,9 @@ export default {
                 if (window.loadMainApp) {
                     return {
                         is: 'router-link',
-                        to: matchingVueRoute.fullPath
+                        to: matchingVueRoute.fullPath,
+                        // Add a `href` attribute to enable native mouse navigation (middle click, ctrl+click, etc.)
+                        href: new URL(matchingVueRoute.fullPath, base).href
```

Comment:
This is likely needed to be moved down to the below block, if it's added here it'll need to have some kind of check since not all `AppLink` components should be openable in another tab.

Reply:
Well currently it only affects converted pages' links to other converted pages (like config-general to config). It's possible there's an edge case I missed.